### ISSUE TITLE
Hinterlands' Verdantine Mobs do not patrol

### ISSUE THREAD

**Issue by Stump876** [azerothcore/azerothcore-wotlk#6216]
Original report from Chromiecraft [https://github.com/chromiecraft/chromiecraft/issues/770](https://github.com/chromiecraft/chromiecraft/issues/770)

#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [x] Alliance
- [x] Horde

##### CONTENT PHASE:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [x] 50-59

##### CURRENT BEHAVIOUR:

Verdantine Mobs (https://wowgaming.altervista.org/aowow/?search=verdantine) are missing patrol patterns.

There are 3 packs where 1 mob moves, all other packs are all stationary in full.

The following mobs dont have any patrol pattern:
GUID 92874, 92875, 92876
![1113](https://user-images.githubusercontent.com/80769357/120708307-b0daf480-c4bb-11eb-86d4-0b6af193eb37.jpg)

GUID 92939, 92938, 92940
GUID 92931, 92930, 92929
GUID 92926, 92927
GUID 92858, 92856, 92857
GUID 93053, 93052, 93051
GUID 92933, 92934, 92932
GUID 92872, 92873, 92871
GUID 93048, 93047, 93049
GUID 92863, 92862
GUID 92861, 92859, 92860
GUID 92870, 92868, 92869
GUID 92867, 92865
GUID 92935, 92936

##### EXPECTED BLIZZLIKE BEHAVIOUR:

The groups of 3 mobs should patrol around walking short distances

##### SOURCE:

https://classic.wowhead.com/npc=12477/verdantine-boughguard#comments
https://youtu.be/Jeze0p0SknY?t=104

##### STEPS TO REPRODUCE THE PROBLEM:

1. Go Creature 9293
2. Explore around and see the packs of mobs do not patrol
3. GUID for each mob listed above

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/40cf7b46d221e562a20264316d660318af213690

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by FrancescoBorzi** [comment]
@Stump876 isn't The Hinterlands' level 40-50? I think the 40-49 label is a better fit

**Comment by FrancescoBorzi** [comment]
@Azcobu isn't The Hinterlands' level 40-50? I think the 40-49 label is a better fit

**Comment by Stump876** [comment]
Yes hinterlands is 40-50, but the mobs themselves are 60-62 elites. A good 10 levels above. That was why I picked a higher level bracket.

**Comment by FrancescoBorzi** [comment]
> Yes hinterlands is 40-50, but the mobs themselves are 60-62 elites. A good 10 levels above. That was why I picked a higher level bracket.

ok well, this is quite a corner case then, and not important at all (as we should fix the problem anyway)
